### PR TITLE
Feature/sign out button

### PR DESCRIPTION
**Pull request by kealanparr**
Hope that the whitespace/style difference aren't too much of an issue

I installed `standard` js style package and ran it against the file I was working on to make sure I was compliant with the styling

Ended up taking a lot of the raw HTML Nunjucks renders from `govUkHeader` and added our feature

### CODE REVIEW

File: packages/web-service/src/pages/applications/applications.njk [hunk 1]
```diff
@@ -13,6 +13,8 @@
   <p class="govuk-body">Return to an existing application by selecting it from your list of licence applications.</p>
 {% endset %}
 
+{% set loggedIn = 'true' %}
```
Comment (GrahamMWillis) [inline packages/web-service/src/pages/applications/applications.njk:16]: This should come down from the handlers as data. You can probably add a flag to additional properties which can be referenced on any page. The code can check for the signed in status from the auth cache.  `request.cache().getAuthData()`

You will probably need to clear the cache in `packages/web-service/src/pages/sign-out/sign-out.js`

**Review by GrahamMWillis — CHANGES_REQUESTED** [review]

File: packages/web-service/src/pages/layout/layout.njk [hunk 2]
```diff
@@ -8,60 +8,103 @@
...
+          float: right;
+          box-sizing: border-box;
+        }
+      </style>
+
+      <div class="govuk-header__content">
+        <a href="/" class="govuk-header__link govuk-header__link--service-name">Apply for a licence to do work that affects a protected species</a>
+
+        {% if loggedIn %}
+          <a class="govuk-header__link signout-link" href="/sign-out">
```
[the remark this answers is not in the document]
Comment (kealanparr) [reply packages/web-service/src/pages/layout/layout.njk:58]: Adressed

File: packages/web-service/src/pages/layout/layout.njk [hunk 3]
```diff
@@ -8,60 +8,103 @@
...
+            </svg>
+            <span class="govuk-header__logotype-text">
+              GOV.UK
+            </span>
+          </span>
+        </a>
+      </div>
+
+      <style>
```
[the remark this answers is not in the document]
Comment (kealanparr) [reply packages/web-service/src/pages/layout/layout.njk:36]: Adressed

File: packages/web-service/src/pages/applications/applications.njk (hunk 1 above)
Comment (GrahamMWillis) [reply packages/web-service/src/pages/applications/applications.njk:16]: Actually you should not need to do it thinking about it. As soon as you unstate the cache is decoupled from the cookie....

Comment (kealanparr) [reply packages/web-service/src/pages/applications/applications.njk]: Addressed

**Review by GrahamMWillis — APPROVED** [review]

**Pull request merged**